docs/immortal-empires/skaven.md

```diff
@@ -0,0 +1,23 @@
+## Throt the Unclean - Clan Moulder
+
+* **Route I - Yes-yes, prepare for war!**
```

Comment:
As mentioned on Discord, I do not really like this route name because it sounds like it is a preparation for the other routes. Each route should feel like a unique and independent campaign.

Reply:
Same with the rest of the flavour regarding this route.